root/static/scripts/common/MB/Control/EditList.js

```diff
@@ -23,7 +23,9 @@ MB.Control.EditList = function (container) {
 
     self.initialize = function () {
         var $voteOptions = $container.find('input[type="radio"]')
-            .first().parents('.voteopts').clone().addClass('overall-vote');
+            .first().parents('.voteopts')
```

Comment:
.find() and .parents() here should probably be moved to their own lines as well.